pyannote/core/annotation.py

```diff
@@ -1423,4 +1425,7 @@ def _repr_png_(self):
         """
 
         from .notebook import repr_annotation
-        return repr_annotation(self)
+        try:
+            return repr_annotation(self)
+        except ImportError:
+            return None
```

Comment:
Would be nice to warn users here that they can get nice visualizations if they do `pip install pyannote.core[notebook]`.